### PR TITLE
contrib/static-web-server: new package (2.32.2)

### PR DESCRIPTION
**Pull request by jcgruenhage**
Still missing a dinit service, but I'd like to add that later. Just using it for local testing at the moment.

### CODE REVIEW

File: contrib/static-web-server/template.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+def post_build(self):
+    self.do(
+        f"target/{self.profile().triplet}/release/static-web-server",
+        env={"SWS_GENERATE": "generated"},
+    )
+
+
+def post_install(self):
+    self.install_license("LICENSE-MIT")
+    self.install_license("LICENSE-APACHE")
```
Comment (nekopsykose) [inline contrib/static-web-server/template.py:28]:
```suggestion
```

File: contrib/static-web-server/patches/0003-generate-man-page-and-completions.patch [hunk 2]
```diff
@@ -0,0 +1,219 @@
+From 6d3747bb733837686df0f0f655bf0cd7ae774346 Mon Sep 17 00:00:00 2001
```
Comment (nekopsykose) [inline contrib/static-web-server/patches/0003-generate-man-page-and-completions.patch:1]: send this upstream and only backport after it's actually merged

**Comment by nekopsykose** [comment]
also put this in user/

**Comment by jcgruenhage** [comment]
Any specific reason for putting this into `user/`?

File: contrib/static-web-server/patches/0003-generate-man-page-and-completions.patch (hunk 2 above)
Comment (jcgruenhage) [reply contrib/static-web-server/patches/0003-generate-man-page-and-completions.patch]: It's not merged *yet*, but I'm happy with just keeping this open until it is merged upstream. Should happen soon.

Comment (jcgruenhage) [reply contrib/static-web-server/patches/0003-generate-man-page-and-completions.patch]: Patch is merged upstream now.

**Comment by jcgruenhage** [comment]
Well with contrib gone and main definitely not being the right place, user it is.

**Pull request merged**